config/manager/kustomization.yaml

```diff
@@ -1,2 +1,14 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
+patches:
+- patch: '[{"op":"add","path":"/spec/template/spec/containers/0/args/-","value":"--target-allocator-image=ghcr.io/open-telemetry/opentelemetry-operator/target-allocator:"}]'
+  target:
+    kind: Deployment
+- patch: '[{"op":"add","path":"/spec/template/spec/containers/0/args/-","value":"--operator-opamp-bridge-image=ghcr.io/open-telemetry/opentelemetry-operator/operator-opamp-bridge:"}]'
+  target:
+    kind: Deployment
```

Comment:
Can you run `make reset` to get rid of this?

Reply:
Doesn't seem to clear this

Reply:
Looks like you added these values without an image version somehow. Could you try resetting this file to `main`, staging the change, and then running `make generate manifests bundle api-docs reset` ?